docs/user/production-considerations/task-manager-production-considerations.asciidoc

```diff
@@ -126,6 +126,20 @@ Throughput is best thought of as a measurements in tasks per minute.
 
 A default {kib} instance can support up to `200/tpm`.
 
+[float]
+===== Automatic estimation
+
+As demonstrated in <<task-manager-health-evaluate-the-capacity-estimation, Evaluate your capacity estimation>>, the Task Manager <<task-manager-health-monitoring, health monitoring>> performs these estimations automatically.
+
+These are estimates based on historical data and should not be used as predictions, but can be used as a rough guide when scaling the system.
+
+We recommend provisioning at least as many {kib} instances as proposed by `proposed_kibana`, but keep in mind that this number is based on the estimated required throughput, which is based on average historical performance, and cannot accurately predict future requirments.
+
+We recommend provisioning enough {kib} instances to ensure a buffer between the estimated `max_throughput_per_minute` and `avg_required_throughput_per_minute`, otherwise there might be insufficient capacity to handle spikes of ad-hoc tasks. How much of a buffer is needed largely depends on your use case, but keep in mind that `avg_required_throughput_per_minute` takes into account recent spikes and, as long as they are representative of your system's behaviour, shouldn't require much of a buffer.
```

Comment:
(suggesting comma after "otherwise")

```suggestion
We recommend provisioning enough {kib} instances to ensure a buffer between the estimated `max_throughput_per_minute` and `avg_required_throughput_per_minute`, otherwise, there might be insufficient capacity to handle spikes of ad-hoc tasks. How much of a buffer is needed largely depends on your use case, but keep in mind that `avg_required_throughput_per_minute` takes into account recent spikes and, as long as they are representative of your system's behaviour, shouldn't require much of a buffer.
```